src/index.html

```diff
@@ -15,10 +15,10 @@
   <body>
     <ul class="tree">
       <li>
-        Fruit
+        <span>Fruit</span>
```

Comment:
you don't need to change index.html

Reply:
<img width="371" alt="image" src="https://github.com/user-attachments/assets/46513cd3-47df-4a51-9004-53268bfa6c51">

Reply:
Hint: you need to wrap each header text into span tag.

Reply:
Ok, I will solve it